.gitattributes

```diff
@@ -3,3 +3,7 @@
 # means that when checking out code, LF endings get converted
 # to CRLF.
 # *.svg text eol=lf
+*.rb   diff=ruby
+*.rake diff=ruby
+*.css  diff=css
+*.html diff=html
```

Comment:
Do you want to add HTML diffs for ERb files too?

Reply:
I thought so too but isn't erb little different (since it can have ruby code inside it)

Reply:
Yeah, but a lot of editors work quite ok if you set ERb's file type to HTML, so I guess it's worth at try.

Reply:
I have added `*.erb  diff=html` now. Thank you @citizen428